aiida/backends/tests/dbimporters.py

```diff
@@ -202,17 +205,15 @@ def test_dbentry_creation(self):
         results = PcodSearchResults([{'id': '12345678'}])
         self.assertEquals(len(results), 1)
         self.assertEquals(str(results.at(0)),
-                          'PcodEntry(license="CC0",'
-                          'db_name="Predicted Crystallography Open Database",'
-                          'version=None,'
+                          'PcodEntry(db_name="Predicted Crystallography Open Database",'
```

Comment:
Do we really want to compare the string representation? I think this is fragile. Better to compare specific values (e.g. `assertEquals(results.at(0).db_name, "Predicted..."` or something equivalent)

Reply:
This is valid for many other places

Reply:
These string representations should be pretty stable. Unless new properties are added, they should stay completely identical.

Reply:
Converted to dictionary comparison. Should be more stable.